lib/lz4hc.h

```diff
@@ -153,11 +154,13 @@ typedef struct
     uint32_t   lowLimit;        /* below that point, no more dict */
     uint32_t   nextToUpdate;    /* index from which to continue dictionary update */
     int        compressionLevel;
-} LZ4HC_CCtx_internal;
+    const LZ4HC_CCtx_internal* dictCtx;
```

Comment:
This increases the size of `LZ4HC_CCtx_internal`.
The risk is, should someone link to this version of the library, 
while allocating a static memory area based on the size of a previous version, 
it may generate out-of-bound read/write when accessing `dictCtx`.

I thought the plan was to refurbish `uint8_t* inputBuffer` instead, preserving the structure size ?

Reply:
Ah, very good. Yes. Let me investigate how to do that.